Library/Homebrew/test/cask/cmd/upgrade_spec.rb

```diff
@@ -222,4 +222,61 @@
       expect(bad_checksum.staged_path).not_to exist
     end
   end
+
+  context "multiple failures" do
+    let(:installed) {
+      [
+        "outdated/bad-checksum",
+        "outdated/local-transmission",
+        "outdated/bad-checksum2",
+      ]
+    }
+
+    before do
+      installed.each { |cask| Cask::Cmd::Install.run(cask) }
+
+      allow_any_instance_of(described_class).to receive(:verbose?).and_return(true)
```

Comment:
Is this just for debugging the tests?

Reply:
I think so. I just added it because it was in the existing tests in the spec file.